airbyte-integrations/connectors/source-chargebee/acceptance-test-config.yml

```diff
@@ -0,0 +1,30 @@
+# See [Source Acceptance Tests](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector/source-acceptance-tests.md)
+# for more information about how to configure these tests
+connector_image: airbyte/source-chargebee:dev
+tests:
+  spec:
+    - spec_path: "source_chargebee/spec.json"
+  connection:
+    - config_path: "secrets/config.json"
+      status: "succeed"
+    - config_path: "integration_tests/invalid_config.json"
+      status: "excefailedption"
+  discovery:
+    - config_path: "secrets/config.json"
+  basic_read:
+    - config_path: "secrets/config.json"
+      configured_catalog_path: "sample_files/configured_catalog.json"
+      validate_output_from_all_streams: yes
+# TODO uncomment this block to specify that the tests should assert the connector outputs the records provided in the input file a file
+#      expect_records:
+#        path: "integration_tests/expected_records.txt"
+#        extra_fields: no
+#        exact_order: no
+#        extra_records: yes
+#  incremental: # TODO if your connector does not implement incremental sync, remove this block
```

Comment:
were you able to run acceptance tests using `./acceptance-test-docker.sh`? I'm curious why you commented out this block.